### PR TITLE
refactor(schematics): migrate from styleext to style

### LINKED ISSUE #2248: Schematics: replace `styleExt` with `style`
With the new version of the Angular CLI, the `styleExt` option from the schematics has been removed. Instead, a `style` option has been added.

We would like to keep our schematics in line with the Angular CLI schematics.

This change includes:

- [ ] Modifying the schematic options
- [ ] Update the schematics code
- [ ] Uppdate the docs

For ref: angular/angular-cli@e92c46a

### PR DESCRIPTION
**Pull request by Jefiozie**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

Migration from `styleext` to `style`

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
When setting the `style` you can do this by `ng set default.styleext`.

Closes #2248 

## What is the new behavior?
With these changes, we are inline with the Angular CLI and you need the following command for setting the style `ng set default.style`

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by Jefiozie** [comment]
cc: @timdeschryver

File: docs/schematics/README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ The `@ngrx/schematics` extend the default `@schematics/angular` collection. If y
 ```json
 "schematics": {
   "@ngrx/schematics:component": {
-    "styleext": "scss"
+    "style": "scss"
```
Comment (timdeschryver) [inline docs/schematics/README.md]: This can be reverted, the docs at `docs/*` are the older docs for v4-6.

**Comment by Jefiozie** [comment]
Okay, I don't exactly know why the bazel tests are failing. Is it because of my changes?

**Comment by Jefiozie** [comment]
Okay, I don't exactly know why the bazel tests are failing. Is it because of my changes?

**Comment by timdeschryver** [comment]
@Jefiozie a re-trigger of the CI solved it

File: docs/schematics/ionic.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ That command will modify the file `.angular-cli.json` so that you don't have to
 - Make SCSS the default style format (the default for Ionic):
 
 ```bash
-ng set defaults.styleExt scss
+ng set defaults.style scss
```
Comment (timdeschryver) [inline docs/schematics/ionic.md]: This can also be reverted

**Comment by Jefiozie** [comment]
Resolved the minor comment

**Comment by timdeschryver** [comment]
@Jefiozie could you also add a breaking change note to the PR please, you can find a template in [CONTRIBUTING.MD](https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#footer)

**Comment by Jefiozie** [comment]
Sure, you also want me to squash the commits?

**Comment by Jefiozie** [comment]
Added the requested breaking change notes.

**Pull request merged**